src/aideck/loaders/yaml.py

```
@@ -0,0 +1,173 @@
+        self.config: YAMLLoaderConfig = config
+
+
+    @classmethod
+    def get_supported_extensions(cls) -> List[str]:
+        """Gets a list of file extensions supported by this loader."""
+        return [".yaml", ".yml"]
+
+    def load(self, source: Union[str, Path, List[Union[str, Path]]]) -> List[Document]:
+        """Loads all YAML documents from the given source synchronously."""
+        try:
+            loop = asyncio.get_running_loop()
```

Ruff (0.13.1)

Local variable `loop` is assigned to but never used

Remove assignment to unused variable `loop`

(F841)